addons/bitbucket/models.py

```diff
@@ -15,9 +16,10 @@
 from framework.auth import Auth
 from osf.models.external import ExternalProvider
 from osf.models.files import File, Folder, BaseFileNode
-from website import settings
 from website.util import web_url_for
 
+settings = importlib.import_module('api.base.settings')
```

Comment:
Why is this necessary?

Reply:
I don't remember the exact error, but this was to get around a circular import when importing node addon settings I believe. I'll get more details!

Reply:
Discussed on FD: using a normal import does cause a circular import.

We haven't figured out why using `import_module` fixes it, but it's a fine workaround for the time being.